src/test/java/com/amihaiemil/eoyaml/YamlIndentationTestCase.java

```diff
@@ -144,4 +147,32 @@ public void readsBadlyIndentedMapping() throws Exception {
             Matchers.equalTo("rultor")
         );
     }
+
+    /**
+     * A badly indented YAML sequence throws an exception.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public void testIndentationAndYamlIndentationException() throws Exception {
```

Comment:
badlyIndentedSequenceThrowsException

All test names should follow the patter xDoesSomething